api/src/main/java/org/apache/brooklyn/api/entity/Entity.java

```diff
@@ -314,4 +361,80 @@
          */
         <T> void emit(Sensor<T> sensor, T value);
     }
+    
+    @Beta
+    public interface PolicySupport {
+        /**
+         * @return an immutable thread-safe view of the policies.
+         */
+        Collection<Policy> getPolicies();
```

Comment:
There is only one method on `Iterable`: `Iterator<T> iterator()`.  And I think it would be nicer if we didn't have `policies().getPolicies()`.  I think the same should be true of `config()` and `sensor()` etc. they would benefit from being an `Iterable`.